swap_meet/clothing.py

```diff
@@ -0,0 +1,18 @@
+from .item import Item
+
+class Clothing(Item):
+    """Class `Item`'s Child class.
+
+    Starts with default values for `condition` and `age`.
+    Initializes with a named attribute for its type, `"Clothing"`.
+    """
+
+    def __init__(self, condition=0, age=0):
+        super()
+        self.category = "Clothing"
+        self.condition = condition
+        self.age = age
```

Comment:
This totally works but there is a tiny opportunity to DRY up your code a little but calling `super().__init__()` with these parameters from the child classes instead. It would be worth trying to rewrite these to utilize that if you can

Reply:
will do that!